lms/static/scripts/frontend_apps/components/InstructorToolbar.tsx

```diff
@@ -23,15 +23,25 @@ export default function InstructorToolbar() {
     gradingEnabled,
   } = instructorToolbar;
 
+  const withGradingControls = gradingEnabled && !!students;
+
   return (
     <header
       className={classnames(
-        'grid grid-cols-1 items-center gap-y-2 p-2',
-        'lg:grid-cols-3 lg:gap-x-4 lg:px-3'
+        'p-2',
+        // Default and narrower screens: content is stacked vertically
+        'grid grid-cols-1 items-center gap-x-4 gap-y-2',
```

Comment:
What's the benefit of using grid over flex here? (Honest question. I get very confused with those two 😅 )

Reply:
`flexbox` is what I reach for by default, and what I use if there is only "one" row or column to lay out. I'll reach for `grid` when there is, well, a grid: multiple rows and columns. In this particular case, the original layout was more grid-dy than it is at present. I'll typically use `grid` if I want to assert more specific control over the dimensions of the items, and `flex` if I want things to, well, flex along a single axis. It's a grey area. As a rule of thumb: if there is a single axis (horizontal along a single row or vertical along a single column) I'll use `flexbox`. If there are multiple axes involved, I'll use `grid`.

Reply:
Thanks!